site/content/en/docs/manual/advanced/slice-and-join.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'CVAT joining and slicing labels'
+linkTitle: 'Joining and Slicing Labels'
+weight: 18
+description: 'This section explains how to slice or join several labels'
+---
+
+In CVAT you can modify labels by either joining multiple labels into
+a single label or slicing a single label into several labels.
+
+This document provides guidance on how to perform these operations effectively.
+
+See:
+
+- [Joining CVAT labels](#joining-cvat-labels)
+- [Slicing CVAT labels](#slicing-cvat-labels)
+
+## Joining CVAT labels
+
+The **Join masks** tool (![Join masks tool icon](/images/join-masks-icon.jpg)),
+is specifically designed to work with mask annotations.
+
+This tool is useful in scenarios where a single object
+in an image is annotated with multiple labels,
+and there is a need to merge these labels into a single, unified label.
+
+![Join masks](/images/joining-tool-01.jpg)
+
+To join masks, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Join masks** ![Join masks tool icon](/images/join-masks-icon.jpg).
+2. Click on the labels, to select masks that you want to join.
+3. (Optional) To remove selection double click on the mask.
```

Comment:
Why double click?
If you double click you will remove selection and select again

Reply:
Weird... I've updated it but changes are not there....